src/java.base/share/classes/java/lang/invoke/LambdaFormEditor.java

```diff
@@ -969,7 +969,7 @@ LambdaForm filterReturnForm(BasicType newType, boolean constantZero) {
             if (newType == V_TYPE)
                 callFilter = null;
             else
-                callFilter = new Name(constantZero(newType));
+                callFilter = new Name(LambdaForm.identity(newType), newType.btWrapper.zero());
```

Comment:
Maybe we could have a `Name` constructor that just takes an `Object` and returns it like this.

Reply:
I don't think that is necessary - the majority of constant values like this are immediately sent to a NamedFunction, which already accepts such constants natively. The only case such a Name is necessary is for LF return values.